src/view/configuration/configuration.jsx

```diff
@@ -580,9 +578,13 @@ const Configuration = ({ formikProps, initInfo }) => {
                                   pattern: currentPattern
                                 })
                                 .then(newPattern => {
-                                  values.instances[
-                                    index
-                                  ].downloadLinkQualifier = newPattern;
+                                  // A bug exists in the Launch UI where the promise is resolved
+                                  // with undefined if the user hits Cancel. Instead, the promise
+                                  // should have never been resolved or rejected.
+                                  // https://jira.corp.adobe.com/browse/DTM-14454
```

Comment:
I tried creating an issue in JIRA for removing this code once DTM-14454 is resolved, but JIRA is struggling. I'll try later.

Reply:
Issue has been added: https://jira.corp.adobe.com/browse/CORE-44498